public/src/js/components/userModalComponent.js

```diff
@@ -7,107 +7,222 @@ var UserModel = require('../models/userModel');
 
 module.exports = React.createClass({
   mixins: [Backbone.React.Component.mixin],
-  
-  attributes: [
-    'idn',
-    'username',
-    'first_name',
-    'last_name',
-    'phone',
-    'website',
-    'github',
-    'is_admin',
-    'is_instructor',
-    'is_student'
-  ],
-
-  componentDidMount: function() {
-    _.each(this.attributes, function (attr) {
-      this.refs[attr].value = this.props.model.get(attr);
-    }, this);
+
+  getInitialState: function() {
```

Comment:
I think this approach can work.  The only real "gotcha", I can see at the moment, would be we have to look out for is overriding computed attributes on the model.

On the userModel something like full_name, would be overridden if we were to simply save the model with this.state.model. We would either be saving with an empty default, or an out-of-sync version.  Either will overwrite the calculated version that is generated in the userModel, if the key full_name in this.state.model comes after the first_name and last_name keys.

Check out my updated PR, In the saveUser function I'm being explicit in drawing out only the attributes we want to include.  

We could switch this up and remove the computed attributes, but I think it is less clear what is actually going on that way.